components/BottomNavigation/examples/BottomNavigationBarControllerExampleViewController.swift

```diff
@@ -15,19 +15,22 @@
 import UIKit
 
 import MaterialComponentsBeta.MaterialBottomNavigationBeta
-import MaterialComponents.MaterialBottomNavigation_ColorThemer
-import MaterialComponents.MaterialBottomNavigation_TypographyThemer
 import MaterialComponents.MaterialColorScheme
 
 class BottomNavigationControllerExampleFixedChildViewController: UIViewController {
 
-  var colorScheme: MDCColorScheming = MDCSemanticColorScheme(defaults: .material201804)
+  var containerScheme: MDCContainerScheming = {
+    let containerScheme = MDCContainerScheme()
+    containerScheme.colorScheme = MDCSemanticColorScheme(defaults: .material201804)
```

Comment:
Tip: I think it's better to just use the default container scheme.

Reply:
Done